contracts/interfaces/IERCHandler.sol

```diff
@@ -25,4 +25,11 @@ interface IERCHandler {
         @return address The {tokenContractAddress} that is currently set for the resourceID.
      */
     function _resourceIDToTokenContractAddress(bytes32 resourceID) external view returns (address);
+
+    /**
+      @notice Set {contractAddress} decimals on dest chain.
+      @param contractAddress Address of contract to be used when making or executing deposits.
+      @param externalDecimals Decimal places of token that is transferred.
+    */
+    function setDecimals(address contractAddress, uint8 externalDecimals) external;
```

Comment:
Don't think we need this public setDecimals.
Though IERCHandler should support setResource function.